src/js/tabs/pid_tuning.js

```diff
@@ -463,7 +463,7 @@ TABS.pid_tuning.initialize = function (callback) {
             const ffInterpolateCheck = $('input[id="ffInterpolateSp"]');
 
             ffInterpolateCheck.prop('checked', FC.ADVANCED_TUNING.ff_interpolate_sp !== 0);
-            $('select[id="ffInterpolate"]').val(FC.ADVANCED_TUNING.ff_interpolate_sp > 0 ? FC.ADVANCED_TUNING.ff_interpolate_sp : 2);
+            $('select[id="ffInterpolate"]').val(FC.ADVANCED_TUNING.ff_interpolate_sp > 0 ? FC.ADVANCED_TUNING.ff_interpolate_sp : 1);
```

Comment:
I suppose this code is only used when we "enable" the FF interpolate to assign some kind of default in the interface.

Can produce some problem for old versions? I suppose now they will see a default of 1 and not 2.

Our defaults are only best effort values, so if this not produces a very bad effect to me is not a problem.

Reply:
@McGiverGim yes it aplies this default if the user disables this feature and want to enable again in the future.

This code doesnt afect to old versions because is a new 10.8.0 code.

Reply:
True, I didn't notice that it was for 1.44...